protobuf/lib/src/protobuf/exceptions.dart

```diff
@@ -4,6 +4,13 @@
 
 part of protobuf;
 
+const _truncatedMessageText = '''
+While parsing a protocol message, the input ended unexpectedly
+in the middle of a field.  This could mean either than the
```

Comment:
No double space between `.field` and `This`.

Last sentence should be "This could either mean that the input has been truncated or that an embedded message misreported its own length."